src/test/librbd/object_map/test_mock_DiffRequest.cc

```diff
@@ -138,6 +138,53 @@ TEST_F(TestMockObjectMapDiffRequest, FastDiffInvalid) {
   ASSERT_EQ(-EINVAL, ctx.wait());
 }
 
+TEST_F(TestMockObjectMapDiffRequest, FastDiffWholeObject) {
```

Comment:
I don't like the test name `FastDiffWholeObject`. Because "whole object" in the CLI just meant that we would going to use fast diff feature, and all these tests in this file for fast diff.

Actually I think we just need to modify the IntermediateDelta to include also `OBJECT_EXISTS` -> `OBJECT_EXISTS_CLEAN` case.